Assignment/solution_student/index.html

```diff
@@ -15,17 +15,69 @@
     <!-- YOUR MARKUP GOES HERE! -->
     <div class="container">
     
-      <header>
-        <h1>FEWD HTML Template</h1>
-      </header>
+      <div id="header">
+        <h1>Relaxr</h1>
+        <nav>
+          <ul>
+            <li><a href="#">About</a></li>
+            <li><a href="#">FAQ</a></li>
+            <li><a href="#">Team</a></li>
+            <li><a href="#">Contact Us</a></li>
+            <li><a href="#">Blog</a></li>
+          </ul>
+        </nav>
+      </div>
 
-      <main>
-        <p>Our HTML starter template!</p>
-      </main>
+      <div id="main">
+        <article>
+          <header>
+            <h1>How I implemented Relaxr in 2 weeks and changed my life.</h1>
+          </header>
+            <img src="images/blog_photo1.jpg">
+             <p><span class="firstcharacter">L</span>orem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+          <footer>
+            Read more >
+          </footer>
+          <hr>
+          <header>
+            <h1>I traveled across the globe thanks to Relaxr’s Excel automation</h1>
+          </header>
+            <img src="images/blog_photo2.jpg">
+             <p><span class="firstcharacter">L</span>orem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+          <footer>
+            Read more >
```

Comment:
Just a minor note: it seems to me that these "Read more >" parts are links that take the user to the full post page, so these should be marked up as `<a>` tags.